src/crypto/public_key/rsa.js

```diff
@@ -210,17 +199,6 @@ function RSA() {
       return key;
     }
 
-    function convertKeyOperation(keyop, errmsg) {
-      return new Promise(function(resolve, reject) {
-        keyop.onerror = function (err) { 
-          reject(new Error(errmsg));
-        }
-        keyop.oncomplete = function (e) {
-          resolve(e.target.result);
-        }
-      });
-    }
-
```

Comment:
Please rebase on top of the current master, as this change would revert IE11 support.